### PR TITLE
Add stadium streetview

### PR DESCRIPTION
**Pull request by aroquev00**
Minor addition, the page now has a panorama Streetview of where we play at local games.
image

### CODE REVIEW

File: aroquev/src/main/webapp/script.js [hunk 1]
```diff
@@ -478,3 +479,16 @@ function createMap() {
       document.getElementById('map'),
       {center: {lat: 23.167, lng: -101.842}, zoom: 5});
 }
+
+let panorama;
+function initializeStadiumSV() {
+  console.log('here');
```
Comment (brettapeters) [inline aroquev/src/main/webapp/script.js:485]: nit. remove `console.log`

File: aroquev/src/main/webapp/script.js [hunk 2]
```diff
@@ -478,3 +479,16 @@ function createMap() {
       document.getElementById('map'),
       {center: {lat: 23.167, lng: -101.842}, zoom: 5});
 }
+
+let panorama;
```
Comment (brettapeters) [inline aroquev/src/main/webapp/script.js:483]: unused?

File: aroquev/src/main/webapp/script.js (hunk 1 above)
Comment (aroquev00) [reply aroquev/src/main/webapp/script.js:485]: Lol forgot that, done.

File: aroquev/src/main/webapp/script.js (hunk 2 above)
Comment (aroquev00) [reply aroquev/src/main/webapp/script.js:483]: True, done.

**Pull request merged**